pypopquiz/video.py

```diff
@@ -43,11 +46,14 @@ def filter_stream(stream: VideoBackend, kind: str, round_id: int, question: Dict
     else:
         raise RuntimeError("Repetition not 1 or multiple 2, got: {:d}".format(repetitions))
 
+    if add_spacer and kind == "question":
+        stream.add_spacer("Get Ready...", duration_s=2)
```

Comment:
Perhaps the spacer can actually be a string instead of a bool (with default the empty string), so that people can easily customize this. But not sure if that would be used a lot.

Reply:
Does empty string mean: no spacer in that case? Or do we need an additional flag?